pkgs/by-name/hy/hyprland-monitor-attached/package.nix

```
@@ -0,0 +1,24 @@
+{ rustPlatform, fetchFromGitHub, lib }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hyprland-monitor-attached";
+  version = "unstable-2023-10-03";
+
+  src = fetchFromGitHub {
+    owner = "coffebar";
+    repo = pname;
```

```suggestion
    repo = "hyprland-monitor-attached";
```
Avoid `repo = pname;`